.github/workflows/ci.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     uses: Dwolla/jenkins-agents-workflow/.github/workflows/build-docker-image.yml@main
     with:
       IMAGE_NAME: dwolla/jenkins-agent-sbt
-      TAG_NAME: CORE_TAG_JDK8_JDK11
```

Comment:
Removing `CORE_TAG_JDK8_JDK11`, in order to just build for jdk11. [Jenkins no longer supports jdk8](https://www.jenkins.io/blog/2022/06/28/require-java-11/), so there is no need to build this agent for both jdk8 and jdk11 anymore.